doc/vmware_workstation_15_5_0_setup.rst

```diff
@@ -0,0 +1,62 @@
+
+For WMware worktation 15.5.0
+============================
+
+If you using WMware not working with latest version of Windows and you need to install old version of WMware. Thing might work a bit difference.
+
+Hence is the steps for running `cortx-va-1.0.2.zip <https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip>`_ for WMware Workstation 15.5.0 `cortx-va-1.0.2.zip <https://github.com/Seagate/cortx/releases/download/VA/cortx-va-1.0.2.zip>`_ in Window 10 version 1890 ( OS build 17763.1637) 
+
+1) Install the `WMware Workstation 15.5.0 <https://www.youwindowsworld.com/en/downloads/virtualization/vmware/vmware-workstation-15-pro/download-535-vmware-workstation-15-pro>`_.
```

Comment:
What is this site for?

if you really want to provide a site to download the VMWare, please provide official side.

Reply:
@huanghua78 unfortunately official site no longer provided old version for download.